server/events/command_runner.go

```diff
@@ -533,6 +533,10 @@ func (c *DefaultCommandRunner) runProjectCmds(cmds []models.ProjectCommandContex
 		case models.ApplyCommand:
 			res = c.ProjectCommandRunner.Apply(pCmd)
 		}
+
+		if res.Error != nil {
+			pCmd.Log.Err("Error running project command %s", res.Error.Error())
```

Comment:
Don't you want to add this to parallel runner as well?

Reply:
hmm i actually might remove this for here and submit as an upstream fix with the parallel runner one as well.